SPM/README.md

```diff
@@ -0,0 +1,11 @@
+## Development setup
+
+Once you have gained access, set up the development environment by following:
+
+1. Clone the [tomtom-navigation-ios-examples] repository
+   ```zsh
+   git clone https://github.com/tomtom-international/tomtom-navigation-ios-examples.git
```

Comment:
nit and might be out of scope 
same as here: 
https://github.com/tomtom-international/tomtom-navigation-ios-examples/pull/45#discussion_r1388156344